kuma/dashboards/jinja2/dashboards/revisions.html

```diff
@@ -193,6 +193,46 @@
         });
     });
 
+    // Send revision to Akismat for Spam or Ham
+    $(document).on('click', '#spam-ham-button', function() {
+        var $this = $(this),
+          $tdObject = $(this).parent(),
+          $trObject = $tdObject.parent(),
+          revisionId = $trObject.data("revisionId"),
+          type = this.value,
+          url = $trObject.data("spamUrl");
```

Comment:
Other variable declarations in this file use multiple ``var``, which I don't like either, but it would be consistent. If you are going to use a different variable assignment pattern, I'd like a 4-char indent so the start of the variable names line up.  This is my amateur JS knowledge though, so feel free to call a nitpick.